plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -419,11 +448,19 @@ synapticsmst_device_enumerate_device (SynapticsMSTDevice *device,
 		break;
 	/* EVB development board */
 	case 0:
+		system = g_strdup (system_type);
+		/* set up GUID
+		 * GUID is MST-$SYSTEMID-$BOARDID
+		 * $BOARDID includes CUSTOMERID in first byte, BOARD in second byte */
+		if (system != NULL)
+			priv->guid = g_strdup_printf ("MST-%s-%u", system, priv->board_id);
 		priv->board_id = (byte[0] << 8 | byte[1]);
 		break;
 	/* unknown */
 	default:
+		#if MSTDBG
```

Comment:
This shouldn't be necessary #if, I think this warning is valid even in the case of non debugging.